gapic-generator-cloud/templates/cloud/client/method/def/_lro.erb

```diff
@@ -46,13 +47,19 @@ def <%= method.name %> request = nil, options: nil, **request_fields
   end
 
   request ||= request_fields
+  # request = Google::Gax::Protobuf.coerce request, to: <%= method.request_type %>
```

Comment:
?

Reply:
This is a future change, I thought I had removed all of them. Removed.